Resources/Prototypes/Recipes/Construction/Graphs/furniture/tables.yml

```
@@ -214,75 +214,57 @@
               
         - to: TableFancyBlack
           steps: 
-            - tag: CarpetBlack
-              name: black carpet
-              icon:
-                sprite: Objects/Tiles/tile.rsi
-                state: carpet-black
+            - material: FloorCarpetBlack
+              amount: 1
+              doAfter: 1
               
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)